### PR TITLE
Passthrough additional params to is_holidays functions

### PR DESCRIPTION
**Pull request by waterkip**
In the Netherlands (and Aruba) some holidays are not for all people, but just
for government employees. Most notable being liberation day. I want to pass the
params to the is_holiday and is_holiday_dt methods to the actual implemenation
so I can get the params hash for `gov => 1, lang => nl` in both functions.

Signed-off-by: Wesley Schwengle <waterkip@cpan.org>

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project, please see the [contribution guidelines](https://github.com/jonasbn/perl-date-holidays/blob/master/CONTRIBUTING.md).
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

You might think, that this is one crazy checklist, but it is just as much written for the maintainer of the involved software :-)

### CODE REVIEW

**Comment by jonasbn** [comment]
Hi @waterkip 

Sounds like a good idea. I will have a look at your PR. 

Thanks

File: lib/Date/Holidays/Adapter/AW.pm [hunk 1]
```diff
@@ -27,8 +29,12 @@ sub is_holiday {
 
     my $holiday;
 
+    my $y = delete $params{year};
+    my $m = delete $params{month};
+    my $d = delete $params{day};
+
```
Comment (jonasbn) [inline lib/Date/Holidays/Adapter/AW.pm:35]: Please write these out for readability and consistency throughout the code.

Meaning `$year` over `$y` and so forth

**Review by jonasbn — CHANGES_REQUESTED** [review]
Everything looks fine, just a minor request for a change

File: lib/Date/Holidays/Adapter/AW.pm (hunk 1 above)
Comment (waterkip) [reply lib/Date/Holidays/Adapter/AW.pm:35]: They have been updated.

**Pull request merged**